odoo/addons/base/models/ir_mail_server.py

```diff
@@ -99,40 +55,6 @@ def extract_rfc2822_addresses(text):
     return [c for c in candidates if is_ascii(c)]
 
 
-def encode_rfc2822_address_header(header_text):
```

Comment:
As the stdlib tooling performs all encoding by default since P3.4, it makes sense to drop the few related helpers we had. They were only used for that precise purpose, and anybody else using them should not need anymore either.
It's worth a very explicit comment about what devs should do now, in the commit that removes it. And it should only land post-v13.

Reply:
The commit message was refering all the related material in the python documentation already and it now provides a few examples.